lib/almodovar.rb

```diff
@@ -16,14 +16,27 @@
 require 'almodovar/to_xml'
 
 module Almodovar
-
   class << self
+    DEFAULT_SEND_TIMEOUT = 30
+    DEFAULT_CONNECT_TIMEOUT = 30
+    DEFAULT_RECEIVE_TIMEOUT = 30
+
     def default_options
-      @default_options ||= {
-        :timeout => 120,
-        :connect_timeout => 30,
-        :user_agent => "Almodovar/#{Almodovar::VERSION}"
+      default = {
+        send_timeout: DEFAULT_SEND_TIMEOUT,
```

Comment:
@AlfonsoUceda I'm wondering if we should respect that symmetry between our own interface and the library that Almovodar uses underneath. Let me explain:

I can't foresee —for now— any use case where we'd need to be this specific. What if we just expose a `timeout` setting (with its proper default) as we did before, to the outside world, but internally we set all the possible timeouts (request, response, etc.) to that same value? Do you see any value on being so granular?